doc/api/cli.md

```diff
@@ -539,6 +539,10 @@ added: v0.1.32
 
 `','`-separated list of core modules that should print debug information.
 
+### `NODE_DEBUG_NATIVE=module[,…]`
+
+`','`-separated list of core C++ modules that should print debug information.
```

Comment:
Should this be _comma-separated_ instead of _`,`-separated_? The former would seem to be a widely-understood term. The latter would seem to be an unusual way to express the same concept.

Reply:
Oh, I see, it's already that way elsewhere in the documentation. Never mind. Consistency is good. We can change it all at once everywhere in the document some other time if desired.